### DISCUSSION TITLE
I found something that looks like syntax error in netrw. Is it an error or just as planned?

### DISCUSSION THREAD

**Discussion by msva** [neovim/neovim#30455] [Q&A] [answered]
Hi there!
Can you advice me, please, is `]` here: 
(0.10) https://github.com/neovim/neovim/blob/release-0.10/runtime/autoload/netrw.vim#L4191
(master) https://github.com/neovim/neovim/blob/master/runtime/autoload/netrw.vim#L4186

is syntax error, or "just as planned"? :thinking:

**Comment by clason** [comment, marked as answer]
It's "just as planned" since it's exactly the same in Vim. If this throws an error for you, please report it at (or make a PR to) vim/vim, where this file is maintained.

(Looks like copypasta to me, but what do I know about Vimscript.)

**Comment by msva** [comment]
ok, thanks.

**Comment by telemachus** [comment]
I'm no Vimscript expert, but that definitely looks wrong to me. I submitted a PR to vim/vim.

https://github.com/vim/vim/pull/15718
